dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +596,32 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  var i = 0;
```

Comment:
You have used `i` as a global variable in the same file. What is the requirement of using another variable with the same name inside a function.

Reply:
changes the variable name